diff --git a/node_modules/naudiodon2/index.js b/node_modules/naudiodon2/index.js
index 9d7a80b..c6c87f0 100644
--- a/node_modules/naudiodon2/index.js
+++ b/node_modules/naudiodon2/index.js
@@ -16,8 +16,8 @@
 const { Readable, Writable, Duplex } = require('stream');
 const portAudioBindings = require("bindings")("naudiodon.node");
 
-var SegfaultHandler = require('segfault-handler');
-SegfaultHandler.registerHandler("crash.log");
+// var SegfaultHandler = require('segfault-handler');
+// SegfaultHandler.registerHandler("crash.log");
 
 exports.SampleFormatFloat32 = 1;
 exports.SampleFormat8Bit = 8;
